src/server.ts

```diff
@@ -2,37 +2,142 @@ import express, { Request, Response, Application, NextFunction } from 'express';
 import { config } from './config';
 import logger from './logger';
 import { v4 as uuid } from 'uuid';
+import axios from 'axios';
+import * as _ from 'underscore';
 
+const url = 'http://www.omdbapi.com?apikey=16d8c738&t=';
```

Comment:
Just `url` is not the best constant name, try to use something more distinct, like `OMDB_BASE_URL`

Reply:
Done